rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/helper/RPAMapperHelper.java

```diff
@@ -16,4 +21,15 @@ public static String prependWithTradingAs(String value) {
     public static boolean isAddressAmended(Party ownParty, TheirDetails oppositeParty) {
         return !ownParty.getAddress().equals(oppositeParty.getAddress()) ? true : false;
```

Comment:
Understood that this isn't a change of yours, but the ternary boolean operator here can just be removed